README.md

```diff
@@ -53,6 +53,23 @@ Optional arguments:
                                    (Defaults to "work")
 
   -h, --help                       Show this message.
+
+Optional ENV:
+
+The following ENV's are likely only utilized when building for the purposes of packaging Kong
+
+  LUAROCKS_INSTALL                 Overrides the `./config --prefix` value (default is `--prefix/luarocks`)
```

Comment:
Style: I think`<prefix>/luarocks` is better. The current syntax is confusing to understand at first, it looks like a command.